scripts/etcd_version_annotations.txt

```diff
@@ -423,6 +423,162 @@ etcdserverpb.WatchResponse.events: ""
 etcdserverpb.WatchResponse.fragment: "3.4"
 etcdserverpb.WatchResponse.header: ""
 etcdserverpb.WatchResponse.watch_id: ""
+grpc.gateway.protoc_gen_openapiv2.options.Contact: ""
```

Comment:
https://github.com/etcd-io/etcd/blob/afade99a7930544a01eb785c681f1431896350b6/tools/proto-annotations/cmd/etcd_version.go#L33

I think we can update the file and remove this change because external package doesn't have etcd version.

Reply:
Good catch, it seems we need to update the script `./scripts/update_proto_annotations.sh` and/or `./tools/proto-annotations/main.go`.

Please feel free to raise a separate PR to fix it; and afterwards I am happy to rebase this PR, thx.

Reply:
It seems that we just need to get `grpc.gateway.protoc_gen_openapiv2.options` added into [externalPackages](https://github.com/etcd-io/etcd/blob/afade99a7930544a01eb785c681f1431896350b6/tools/proto-annotations/cmd/etcd_version.go#L32).

@fuweid Please feel free to deliver a PR to fix it. thx

cc. @serathius

Reply:
Just resolved in this PR.